### DISCUSSION TITLE
什么时候会发布2.0.3？ 修复2.0.2的bug

### DISCUSSION THREAD

**Discussion by DA1OOO** [apache/doris#26497] [RFC]
什么时候会发布2.0.3？ 修复2.0.2的bug

**Comment by LemonLiTree** [comment]
expected this month

**Comment by zhbdesign** [comment]
> 什么时候会发布2.0.3？ 修复2.0.2的bug

什么bug

**Reply by DA1OOO** [reply]
https://github.com/apache/doris/issues/25644

**Reply by ixzc** [reply]
2.0.3 rc02 https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-2.0.3-rc02-bin-x64.tar.gz
Release the official version of doris-2.0.3 in about 2 weeks

**Reply by DA1OOO** [reply]
@ixzc thanks, but why the official version of doris-2.0.3 still not release now.

**Reply by ixzc** [reply]
soon

**Comment by rulerhuang** [comment]
月底了，来催更 2.0.3 release

**Reply by DA1OOO** [reply]
+1

**Reply by LemonLiTree** [reply]
soon

**Reply by bingwill** [reply]
+1
